client/my-sites/plan-features/style.scss

```diff
@@ -1,3 +1,33 @@
+$header-banner-height: 20px;
```

Comment:
lets move this under the .plan-features scope so the sass variable name is not global